plugin/src/withIosAppcuesRichPush.ts

```diff
@@ -87,22 +114,14 @@ const withAppcuesXcodeProject: ConfigPlugin<ConfigProps> = (config, props) => {
       APPCUES_NSE_TARGET.NAME
     );
 
-    // Add the group to the project group. This makes the files appear in the file explorer in Xcode.
-    // Searching for the nameless group is a proxy for the root group.
-    const groups = xcodeProject.hash.project.objects['PBXGroup'];
-    for (const groupUUID of Object.keys(groups)) {
-      if (
-        typeof groups[groupUUID] === 'object' &&
-        groups[groupUUID].name === undefined
-      ) {
-        xcodeProject.addToPbxGroup(
-          appcuesNotificationServiceGroup.uuid,
-          groupUUID
-        );
-        // There should only be a single nameless group, but break anyways just to be sure it's not added twice.
-        break;
-      }
-    }
+    // Add the group to the main group. This makes the files appear in the file explorer in Xcode.
+    const mainGroupUUID =
+      xcodeProject.pbxProjectSection()[xcodeProject.getFirstProject().uuid]
+        .mainGroup;
+    xcodeProject.addToPbxGroup(
+      appcuesNotificationServiceGroup.uuid,
+      mainGroupUUID
+    );
```

Comment:
This does the exact same thing as this block did previously, just optimized inspired by the implementation here: https://github.com/expo/config-plugins/blob/4eb10b779422fe96139fe47dbce6a6de20e4e398/packages/ios-stickers/src/xcodeSticker.ts#L7-L19